src/Logs.js

```diff
@@ -18,123 +22,161 @@ function Logs(props) {
   }
 
   function changeDayBackgroundIfHasLog(date, view) {
-    let dayHasLog = retrieveLogDatabase().hasOwnProperty(moment(date).format('DD MM YYYY'))
+    const dayHasLog = retrieveLogDatabase().hasOwnProperty(moment(date).format('DD MM YYYY'));
     if (view === 'month' && dayHasLog) {
       return 'dayWithLog';
     }
   }
 
   function changeDayBackgroundIfFuture(date, view) {
     const isInFuture = moment().isBefore(date);
-    console.log(isInFuture)
 
     if (view === 'month' && isInFuture) {
       return 'dayInFuture';
     }
   }
 
-  return(
-    <div style={{display: 'flex', flexDirection: 'row', alignItems: 'center'}}>
+  return (
+    <div style={{ display: 'flex', flexDirection: 'column', alignItems: 'center' }}>
       <Calendar onClickDay={handleClickDay} tileClassName={({ activeStartDate, date, view }) => [changeDayBackgroundIfHasLog(date, view), changeDayBackgroundIfFuture(date, view)]} />
       <LogScreenButtons currentScreen={setcurrentTab} />
       {currentTab}
     </div>
-  )
+  );
 }
 
 function LogScreenButtons(props) {
   const [clickedDay] = useContext(ClickedDayContext);
-  let dayHasLog = retrieveLogDatabase().hasOwnProperty(moment(clickedDay).format('DD MM YYYY'));
-  let isInFuture = moment(clickedDay).isAfter();
+  const dayHasLog = retrieveLogDatabase().hasOwnProperty(moment(clickedDay).format('DD MM YYYY'));
+  const isInFuture = moment(clickedDay).isAfter();
 
   if (isInFuture) {
-    return <div></div>;
-  } else if (!dayHasLog) {
+    return <div />;
+  } if (!dayHasLog) {
     return (
       <div className="logScreenButtons">
         <Button className="logScreenButton" onClick={() => props.currentScreen(<LogScreenCreate />)}>Create Log</Button>
       </div>
-    )
-  } else if (dayHasLog) {
-      return (
-        <div className="logScreenButtons">
-          <Button className="logScreenButton" onClick={() => props.currentScreen(<LogScreenRemove />)} currentScreen={props.currentScreen}>Remove Log</Button>
-          <Button className="logScreenButton" onClick={() => props.currentScreen(<LogScreenView />)}>View log</Button>
-        </div>
-  )
+    );
+  } if (dayHasLog) {
+    return (
+      <div className="logScreenButtons">
+        <Button className="logScreenButton" onClick={() => props.currentScreen(<LogScreenRemove />)} currentScreen={props.currentScreen}>Remove Log</Button>
+        <Button className="logScreenButton" onClick={() => props.currentScreen(<LogScreenView />)}>View log</Button>
+      </div>
+    );
   }
 }
 
-function LogScreenCreate(props) {
+function LogScreenCreate() {
```

Comment:
Same idea on naming here. You are in the Logs.js file, you don't need to call this function `LogScreenCreate` you can just call it `form` or something